packages/ckeditor5-dev-release-tools/tests/utils/checkversionavailability.js

```diff
@@ -34,7 +34,7 @@ describe( 'dev-release-tools/utils', () => {
 		} );
 
 		it( 'should resolve to true if version does not exist (npm >= 8.13.0)', () => {
-			stubs.shExec.rejects( new Error( 'npm ERR! code E404' ) );
+			stubs.shExec.rejects( new Error( 'code E404' ) );
```

Comment:
Could we please split this into two cases?

1. npm >= 8.13 && < 10
2. npm >= 10

I want to ensure we support both npm responses (even if the same line handles it).

Reply:
Addressed in [1ebd23a](https://github.com/ckeditor/ckeditor5-dev/pull/967/commits/1ebd23a497b4cd2da9b7b03c0e43dfd75c0090f0)